services/scraper/scraper_test.go

```diff
@@ -0,0 +1,126 @@
+package scraper_test
+
+import (
+	"github.com/hoangmirs/go-scraper/models"
+	"github.com/hoangmirs/go-scraper/services/scraper"
+	"github.com/hoangmirs/go-scraper/tests/fabricators"
+	. "github.com/hoangmirs/go-scraper/tests/test_helpers"
+
+	"github.com/beego/beego/v2/client/orm"
+	"github.com/bxcodec/faker/v3"
+	"github.com/dnaeon/go-vcr/recorder"
+	"github.com/gocolly/colly/v2"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ScraperService", func() {
+	AfterEach(func() {
+		TruncateTables("keyword_result")
+	})
+
+	Describe("#Run", func() {
+		Context("given valid attributes", func() {
+			It("gets the result", func() {
```

Comment:
How about the following as it matches the naming of the struct `ParsingResult`:

```suggestion
			It("returns the parsing result", func() {
```